pkg/query/storeset.go

```diff
@@ -36,13 +37,14 @@ type StoreSpec interface {
 }
 
 type StoreStatus struct {
-	Name      string
-	LastCheck time.Time
-	LastError error
-	Labels    []storepb.Label
-	StoreType component.StoreAPI
-	MinTime   int64
-	MaxTime   int64
+	Name             string
+	LastCheck        time.Time
+	LastHealthyCheck *time.Time
```

Comment:
Any reason this is a pointer?

Reply:
I figured if it's a pointer I can check if it's nil and has been set before ? Not sure if it's the best approach in Go ?

Anyway, it's gone now ! Thanks